### ISSUE TITLE
IP: Prevalidation, add criteria check on CSV headers

### ISSUE THREAD

**Issue by maxvisser** [teamforus/general#265]
Learn more about improvement proposals: https://bit.ly/2xLJT3R

## Impacted areas:

- [ ] Design
- [ ] Code
- [ ] Infrastructure
- [ ] Users
- [ ] Finances
- [ ] Processes

## Current situation
This development is to improve the administrative handling of the upload. At this time, the CSV would need some key/values which are for the bulk the same. For people who have equal rights, most of these columns will have the same content and are therefore redundant. 

## Desired situation
A nice-to-have is to prefill these columns by defining them once and this is done automatically. Enhancing the current implementation of adding _eligible key/value by selecting a fund.

## Plan
Make a modal after pressing upload where you showcase which keys are in the CSV; being able to see which keys are missing which are needed for the fund criteria and let user fill in this data. Prefill value with desired fund criteria value

**Comment by jamalv** [comment]
@maxvisser if i understand this correctly it will help with a less weird UX for uploading a CSV, right? (good thing!) But there are no functional changes? I therefore downgraded this to a should have.

**Comment by maxvisser** [comment]
Yes UX would be improved. The fund selection would also be less important.

**Comment by jamalv** [comment]
Ah interesting, les important or redundant?

**Comment by maxvisser** [comment]
The only reason for it is if you want to make duplicates for BSN's (as only the BSN is unique per fund);

**Comment by jamalv** [comment]
It seems like this is not needed for current projects' so downgrading this to could-have and putting it on hold

**Comment by maxvisser** [comment]
On hold so I will unassign myself.
